_minutes/2024-12-05-wecg.md

```diff
@@ -0,0 +1,178 @@
+ * [sohail] Just an FYI.
+ * [tomislav] What's the stage internally for Chromium?
+ * [oliver] I believe Sohail from Microsoft are proposing to implement this.
+ * [mukul] I believe we heard from Chrome that they were open to it but didn't have the bandwidth to do it, so we're taking it on.
+
+[Issue 732](https://github.com/w3c/webextensions/issues/732): Proposal: Add permanentNodeType property to BookmarkTreeNode
+
+ * [oliver] Looking further into the future, looking at changes to bookmark folders. Before making changes we'd like to look at use cases that are useful to extensions to make it easier to manage bookmarks. Current (Chrome) extension developers look at the ID. There are edge cases where the ID can change. Devs fall back to sting matching. In Firefox the ID isn't numerical. This proposal introduces a new property to identify what are special nodes and what type they are. James and Monica are here from Chrome, they're working on the bookmark changes.
+ * [monica] Thanks oliver, no additional notes at the moment.
+ * [rob] Is there any documentation about the upcoming changes?
+ * [oliver] The other issue (733) points to a blog post.
+ * [james] Probably a bit early to share beyond what's in the public post.
+ * [monica] To give a general idea, today all bookmarks are local are synced. We're introducing another model where some are local, some are account synced. There may be more than one bookmark bar folder, one for local account and one for synced content.
+ * [simeon] Is is possible for items to be interposed?
+ * [monica] In the extension API they will be separate.
+ * [rob] Would permanentNodeType be inherit by bookmarks, e.g. BOOKMARKS_BAR on individual bookmarks, or just the folder at the top level?
+ * [monica] The folders are typed, the nodes inside them are normal, so they need to be separate.
+ * [rob] As an extension developer I've had to parse the content of a bookmark to figure out it's type, so having an explicit field would be welcome. Oliver, you mentioned extensions parsing the ID - do you have an analysis of use cases to share? I'd like to make sure that what we design matches with actual uses.
+ * [oliver] Nothing off hand, but we can share examples of what we've seen in the wild. It seemed like Chrome, Edge, and Safar had similar concepts for what is synced.
+ * [timothy] We don't currently support the Bookmarks API so we'd need to look into it. I don't have details on all of the special cases that we have, but what's been discussed so far sounds good. We should be able to work with it.
+ * [tomislav] I think I heard two different things from Oliver and James/Monica. Are we trying to standardize types or are we expecting that they will be browser specific?
+ * [monica] This is what we have in Chrome. Oliver?
+ * [oliver] FOr something like the bookmarks bar, it would be nice to be consistent across browsers. Can have additional discussions about what we do for concepts that are more browser specific.
```

Comment:
```suggestion
 * [oliver] For something like the bookmarks bar, it would be nice to be consistent across browsers. Can have additional discussions about what we do for concepts that are more browser specific.
```